changelog/spike-code-splitting-intl-input

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: add
+
+Code splitting the intl-tel-input library and load it on demand
```

Comment:
Can we improve the message, please? Because we are reducing a bundle size by moving phone input into a separate one - how do we describe in code, but wah do we do - this is in this log entry.